packages/fetchai/connections/p2p_libp2p/aea/api.go

```diff
@@ -159,11 +170,14 @@ func (aea *AeaApi) Init() error {
 	logger.Debug().Msgf("uri public: %s", uri_public)
 	logger.Debug().Msgf("uri delegate service: %s", uri_delegate)
 
-	if aea.msgin_path == "" || aea.msgout_path == "" || aea.id == "" || uri == "" {
+	if aea.id == "" || uri == "" {
 		err := errors.New("couldn't get AEA configuration")
 		logger.Error().Str("err", err.Error()).Msg("")
 		return err
 	}
+	if aea.msgin_path == "" || aea.msgout_path == "" || aea.agent_addr == "" {
+		aea.standalone = true
+	}
```

Comment:
why not and? If I accidentally forget to provide one field it will run fine in standalone mode but I probably prefer it to fail?

Reply:
Good point